torch/csrc/jit/autodiff.cpp

```diff
@@ -134,25 +86,15 @@ bool isDifferentiable(Node* n) {
     return n->get<bool>(attr::train).value();
   }
 
-  auto schema = n->maybeSchema();
-  if (schema && hasGradientInfoForSchema(*schema)) {
-    return true;
-  }
-
   if (n->matches(
           "aten::expand(Tensor self, int[] size, *, bool implicit) -> Tensor")) {
     return n->get<std::vector<int64_t>>(attr::size) &&
-        n->is_constant(attr::implicit) &&
-        n->namedInput(attr::self)->type()->cast<CompleteTensorType>();
```

Comment:
why we remove this complete tensor check?

Reply:
It's not necessary to be CompleteTensorType. See the conversation in jit-perf-dev ;)